src/util/c_types.cpp

```diff
@@ -142,6 +142,26 @@ type2tc long_long_int_type2()
   return get_int_type(config.ansi_c.long_long_int_width);
 }
 
+typet int128_type()
+{
+  return signedbv_typet(128);
```

Comment:
Sorry, have missed this. Should we use the new config setting `int_128_width` here and in the functions below?